amp.hbs

```diff
@@ -1110,6 +1123,9 @@
     {{/if}}
     <main class="content" role="main">
         <article class="post">
+         {{#has tag="#deprecated"}}
+                <div class="disclamer-deprecated-post">Disclaimer: This blog post has been marked as deprecated, therefore some of the content might be out of date.</div>
+              {{/has}}
```

Comment:
This would benefit from proper indenting

Reply:
Also, this would only work if you add the deprecated tag to the post, but posts show this disclaimer also automatically, couldnt we add this also if the post date is less than 2 years? not sure how, but something like

```hbs
{{#with (sub (now) date) as timeDifference}}
    {{#with (divide timeDifference 31536000000) as years}}
        {{#if (gt years 2)}}
            <div class="disclaimer-deprecated-post">Disclaimer: This blog post has been marked as deprecated, therefore some of the content might be out of date.</div>
        {{/if}}
    {{/with}}
{{/with}}
```